### ISSUE TITLE
[Enhance]Add support for importing bangumi records with one click.

### ISSUE THREAD

**Issue by biqiguhexuenai** [UweTrottmann/SeriesGuide#1034]
Add support for importing bangumi records with one click.It's useful for people who watch fanfares.

**Comment by UweTrottmann** [comment]
Thanks! Do you have actual details on how to do this (are you a software developer?) or is this just a general suggestion?

**Comment by biqiguhexuenai** [comment]
> Thanks! Do you have actual details on how to do this (are you a software developer?) or is this just a general suggestion?

No, I'm not a developer, but there are projects on github that you can refer to.
https://github.com/czy0729/Bangumi
https://github.com/xiaoyvyv/bangumi

**Comment by biqiguhexuenai** [comment]
> Thanks! Do you have actual details on how to do this (are you a software developer?) or is this just a general suggestion?

For example, you can import your favourite shows, favourites, history, etc. just like trakt.

**Comment by UweTrottmann** [comment]
To be honest, likely won't add another service.

But if anyone wants to write a script or similar, these are the steps required:
- export shows with titles (no TMDB ID seems to be linked)
- search for each title on TMDB, possibly ask user to confirm
- append to an existing or create a new SeriesGuide JSON shows backup file

**Comment by biqiguhexuenai** [comment]
> To be honest, likely won't add another service.
> 
> But if anyone wants to write a script or similar, these are the steps required:
> 
> * export shows with titles (no TMDB ID seems to be linked)
> * search for each title on TMDB, possibly ask user to confirm, obtain the TMDB ID
> * append to an existing or create a new SeriesGuide JSON shows backup file

Okay, thanks for your patience.
